main.js

```diff
@@ -71,7 +71,7 @@ function createWindow() {
 }
 
 global.dataPath = app.getPath('userData');
-global.isDev = false;
+global.isDev = true;

```

Comment:
it's not going to write to the sheet in production, set `global.isDev = dev;` for automatic environment detection.